### ISSUE TITLE
Discussion about license

### ISSUE THREAD

**Issue by PANDASANG1231** [UBC-MDS/featurescreator#36]
Hey guys, let's discuss our choice of license. Right now, we are using MIT License created by tools in default. Do you think it's fine to keep using that?

What do you guys think about it? @NikitaShymberg  @SonQBChau  @veerupandey

**Comment by NikitaShymberg** [comment]
I think an MIT license is good because it's very permissive! It explicitly states that we have no warranty which is great. And it allows others to use our package as long as they include a copy of the license. I think that for this project that is perfect.

**Comment by PANDASANG1231** [comment]
Yes, I agree. Besides, MIT License is not only welcoming to open-source developers, but also to businesses. It might be helpful in the future.

**Comment by veerupandey** [comment]
I agree that the MIT license is the most appropriate for our package. We need to keep improving the packages we've built, and the MIT License will encourage more developers to contribute.

**Comment by veerupandey** [comment]
We should probably avoid GPL/AGPL/LGPL and MPL licences because we don't want derivative works of this package licenced under the same open source licence. Again, the MIT licence is a great fit for our project because it allows us to create a derivative work under the developer's preferred licence along with other reasons mentioned by @NikitaShymberg  and @PANDASANG1231.

**Comment by SonQBChau** [comment]
I think we should also consider BSD license as the 2 packages we depend on `Numpy` and `Pandas` using BSD. It's basically MIT except if someone forks our work, they have to mention the original authors.
